### PR TITLE
Map update followup

### PR DESCRIPTION
**Pull request by zhut19**
**What is the problem / what does the code in this PR do**
After merging https://github.com/XENONnT/private_nt_aux_files/pull/50, in which we split the S2 sum-PMT LCE map out into an independent file and normalize to 1, so that we can keep the s2_secondary_sc_gain as the data-driven value in the unit of  PE (observed) per electron.

We also decide that s2_secondary_sc_gain value contains the factor '1+p_double_pe_emision', thus when calculating the S2 gain we always divide it by '1+p_double_pe_emision'

### CODE REVIEW

**Review by petergaemers — APPROVED** [review]

File: wfsim/load_resource.py [hunk 1]
```diff
@@ -268,3 +263,29 @@ def reduce_last_dim(self):
         shape[-1] = 1
 
         return DummyMap(const, shape)
+
+
+def clear_cached_configs():
```
Comment (ramirezdiego) [inline wfsim/load_resource.py:268]: @zhut19 Is this function used?

Comment (zhut19) [reply wfsim/load_resource.py:268]: Ah, this one is somehow in my local master branch, keep mixing into my or. It's not used anywhere.

**Comment by ramirezdiego** [comment]
@petergaemers You tested with a local copy of the private files, right? I think the new normalized map wasn't uploaded to the database yet, but maybe I am doing something wrong in my context.

File: wfsim/load_resource.py (hunk 1 above)
Comment (ramirezdiego) [reply wfsim/load_resource.py:268]: Ok, will you delete it or want to keep it there for an upcoming cache cleanup PR?

Comment (zhut19) [reply wfsim/load_resource.py:268]: Actually, I don't see if this will be used officially in the future. Let me delete it.

**Review by ramirezdiego — APPROVED** [review]
Looks good, I simulated some SEs and the size makes sense now :)

PS: The new map still needs to make it to the database, unless I did something wrong with my environment (see comment above to Peter).

PPS: I think it would make sense to make a new patch release, 0.5.1, based only on this change. Just because it resolves the "bug" on the S2 area with default configurations.

**Comment by petergaemers** [comment]
I've updated the database

**Pull request merged**